src/lib/config.ts

```diff
@@ -1290,6 +1290,11 @@ export class default_config {
      * https://fusejs.io/api/options.html#threshold
      */
     completionfuzziness = 0.3
+
+    /**
+     * Whether to show article url in the document.title of Reader View.
+     */
+    urlinreaderviewtitle: "true" | "false" = "false"
```

Comment:
Tridactyl (mostly) has a pseudo-namespacing where the category/noun comes first then the verb, so `:tabopen` rather than `:opentab`. It makes it easier to find things with completions

for this I guess `readerurlintitle` would be ok?